src/types/types.ts

```diff
@@ -12,6 +12,7 @@ export interface Action {
   name: string;
   synchronization: boolean;
   documentation: string;
+  color: [number, number, number, number] | null; // rgba color, [0, 255]
```

Comment:
As I commented in the other PR, consider making this values in [0, 1] and do the conversion when setting the color.